tests/preProcessor_test.py

```diff
@@ -77,6 +77,45 @@ def test_inplace_no_remember_curve_type(self, FontClass):
             assert CURVE_TYPE_LIB_KEY not in ufo.layers.defaultLayer.lib
             assert glyph_has_qcurve(ufo, "c")
 
+    def test_custom_filters(self, FontClass):
+        ufo1 = FontClass(getpath("TestFont.ufo"))
+        ufo1.lib[FILTERS_KEY] = [
+            {"name": "transformations", "kwargs": {"OffsetX": -40}, "pre": True}
+        ]
+        ufo2 = FontClass(getpath("TestFont.ufo"))
+        ufo2.lib[FILTERS_KEY] = [{"name": "transformations", "kwargs": {"OffsetY": 10}}]
+
+        glyphSets0 = TTFPreProcessor(ufo1).process()
+        glyphSets1 = TTFPreProcessor(ufo2).process()
+
+        assert (glyphSets0["a"][0][0].x - glyphSets1["a"][0][0].x) == -40
+        assert (glyphSets1["a"][0][0].y - glyphSets0["a"][0][0].y) == 10
+
+    def test_custom_filters_as_argument(self, FontClass):
+        ufo1 = FontClass(getpath("TestFont.ufo"))
+        ufo2 = FontClass(getpath("TestFont.ufo"))
+        filter1 = loadFilterFromString("RemoveOverlapsFilter(backend='pathops')")
```

Comment:
why loading from string? There ought to be separate unit tests for that, here you're only testing that you can pass a `filters` argument